std/js/html/midi/Input.hx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C)2005-2019 Haxe Foundation
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+// This file is generated from mozilla\MIDIInput.webidl. Do not edit!
+
+package js.html.midi;
+
+/**
+	Use the `MIDIInput` interface of the Web MIDI API to access and pass messages to a MIDI input port.
+
+	Documentation [MIDIInput](https://developer.mozilla.org/en-US/docs/Web/API/MIDIInput) by [Mozilla Contributors](https://developer.mozilla.org/en-US/docs/Web/API/MIDIInput$history), licensed under [CC-BY-SA 2.5](https://creativecommons.org/licenses/by-sa/2.5/).
+
+	@see <https://developer.mozilla.org/en-US/docs/Web/API/MIDIInput>
+**/
+@:native("MIDIInput")
+extern class Input extends Port
```

Comment:
Damn this is even worse than `Access` I commented before :)

Reply:
Good point on collisions, I think we should break convention here and keep the MIDI prefix – I'll update the PR

It's not such a strict convention since some modules keep their prefixes when they're not uniform like `html.eme` or `html.audio`.

I'd be keeping prefixes for all submodules however I think this would affect too much existing code